### ISSUE TITLE
Transformers version 

### ISSUE THREAD

**Issue by SuryaThiru** [zlwang-cs/LASER-release#4]
Hi with `transformers==4.3.2`, I'm getting the following error:

```
[/home/surya/NEU/Research/VDU/vdu_experiments/libs/LASER-release/env/lib/python3.10/site-packages/tqdm/auto.py:21](https://file+.vscode-resource.vscode-cdn.net/home/surya/NEU/Research/VDU/vdu_experiments/libs/LASER-release/env/lib/python3.10/site-packages/tqdm/auto.py:21): TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html
  from .autonotebook import tqdm as notebook_tqdm
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[1], line 17
     15 from s2s_ft import utils
     16 from s2s_ft.config import BertForSeq2SeqConfig
---> 17 from s2s_ft.modeling import LayoutlmForSequenceToSequence
     19 from s2s_tag.config import *
     20 from s2s_tag.utils import *

File [~/NEU/Research/VDU/vdu_experiments/libs/LASER-release/s2s_ft/modeling.py:13](https://file+.vscode-resource.vscode-cdn.net/home/surya/NEU/Research/VDU/vdu_experiments/libs/LASER-release/~/NEU/Research/VDU/vdu_experiments/libs/LASER-release/s2s_ft/modeling.py:13)
     10 import torch.nn.functional as F
     11 from transformers import BertConfig
---> 13 from transformers.modeling_bert import \
     14     BertPreTrainedModel, BertSelfOutput, BertIntermediate, BertOutput, BertPredictionHeadTransform
     15 from transformers.modeling_roberta import ROBERTA_PRETRAINED_MODEL_ARCHIVE_MAP
     16 from transformers.modeling_bert import BERT_PRETRAINED_MODEL_ARCHIVE_MAP

ModuleNotFoundError: No module named 'transformers.modeling_bert'
```

It seems these modules were changed in an older version. Could you check the exact version of transformers used?

**Comment by zlwang-cs** [comment]
Hi,

Thanks for your interest in our work. Yeah, the `transformers` package has been updated several times. Can you try `transformers==2.10.0`?

```
$ python
Python 3.8.5 (default, Sep  4 2020, 07:30:14)
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import transformers
2023-05-09 14:06:35.744426: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.11.0
>>> transformers.__version__
'2.10.0'
>>>
```

**Comment by SuryaThiru** [comment]
`pip install transformers==2.10.0`

Gives a error with rust version:

```
      91 | #![cfg_attr(feature = "nightly", feature(const_fn))]
         |                                          ^^^^^^^^ feature has been removed
         |
         = note: split into finer-grained feature gates

      For more information about this error, try `rustc --explain E0557`.
      error: could not compile `lock_api` (lib) due to previous error
```

Do you have the rust version you used while installing `transformers`? `tokenizers` installation require the right version of rust.

**Comment by zlwang-cs** [comment]
Hi,

My server didn't install Rust. So I have no idea about this bug. I think you should create a new Python environment to make sure the dependency problem is solved. If this doesn't fix the problem, I guess the problem comes from the server side.

**Comment by RudreshVeerkhare** [comment]
@zlwang-cs can you tell what was the python version installed in your server?

@SuryaThiru I tried with Python 3.7 so was able to install without requiring rust compiler, because when I initially tried it with Python 3.10, I was getting the same error

**Comment by zlwang-cs** [comment]
@RudreshVeerkhare Hi, thanks for your help to fix the issue! My python is 3.8.5.

@SuryaThiru As I mentioned, I don't know why Rust is involved here. I believe it could be the python environment issues. Please try the solution Rudresh suggested.

**Comment by SuryaThiru** [comment]
I will try a different python version. I believe hugging face fast tokenizers use rust.
